test/utilities.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2017, Lakshman Anumolu, Pradeep Garigipati
+ * All rights reserved.
+ *
+ * This file is part of MeshIO whose distribution is governed by
+ * the BSD 2-Clause License contained in the accompanying LICENSE.txt
+ * file.
+ */
+
+#include <meshio/vectors.hpp>
+#include <meshio/utilities.hpp>
+
+#include <gtest/gtest.h>
+
+using namespace std;
+using namespace meshio;
+
+TEST(VECTORS, MISCELLANEOUS)
+{
```

Comment:
May be split tests into `TEST(VECTORS, 2D)` , `TEST(VECTORS, 3D)` and `4D` if possible. Better readable and easy to debug and when they fail tests.